packages/@aws-cdk/aws-glue/README.md

```diff
@@ -448,3 +448,88 @@ new glue.Table(this, 'MyTable', {
 | array(itemType: Type)               	| Function 	| An array of some other type                                       	|
 | map(keyType: Type, valueType: Type) 	| Function 	| A map of some primitive key type to any value type                	|
 | struct(collumns: Column[])          	| Function 	| Nested structure containing individually named and typed collumns 	|
+
+## View
```

Comment:
```suggestion
## Athena View
```

I'm **tentatively** okay with this being called a `glue.View` but I think we should make an important distinction throughout the README and docs. There is no such thing as a glue view (right?). We should always call it the same thing, like "glue athena view" or "glue integration with athena views". We are going to have to hammer that home in order to avoid confusing users.